content/applications/general/email_communication/faq.rst

```diff
@@ -1,174 +1,208 @@
-of what can be helpful:
+of what can be helpful when reaching out to support about an issue:
+
 
-- The **EML** of the email, stating for *Electronic Mail*, is the file format containing all the
-  technical information required for an investigation. The documentation of your own email provider
-  might help you on how to get your EML files. Once you get the EML of the email, adding it
-  in the attachment of your ticket is the most efficient way for us to investigate. The support
-  will mainly focus on redundant issues.
+The **EML** (or **headers**) of the email is the file format containing all the technical
+information required for an investigation. The documentation from the email provider might provide
+the process on how to get EML/header files. Once the headers of the email are obtained, adding it
+into the Odoo support ticket is the most efficient way for the Odoo Support team to investigate.
 
   .. seealso::
      - `Gmail documentation
        <https://support.google.com/mail/answer/29436>`_
      - `Outlook documentation
        <https://support.microsoft.com/en-us/office/view-internet-message-headers-in-outlook-cd039382-dc6e-4264-ac74-c048563d212c#tab=Web>`_
 
-- The exact flow you are following in order to normally receive those emails in Odoo. Here are
-  examples of questions whose answers can be useful:
-
-  - Is this simply a reply from an email going out from Odoo ?
-
-  - Are you using an incoming email server or somehow redirecting?
+The exact flow that is being followed in order to normally receive those emails in Odoo. Here are
+examples of questions whose answers can be useful:
 
-  - Can you provide us with an example of an email that has been correctly forwarded ?
+- Is this simply a reply from an email going out from Odoo?
+- Is there an incoming email server being used, or is the email somehow being redirected?
+- Is there an example of an email that has been correctly forwarded?
 
-- Providing answers to the following questions:
+Providing answers to the following questions:
 
-  - Is it a generic issue or is it specific to a use case? If yes, which one exactly?
+- Is it a generic issue, or is it specific to a use case? If specific to a use case, which one
+  exactly?
+- Is it working as expected? In case the email is sent using Odoo, the bounce email should reach the
+  Odoo database, and display the :ref:`red envelope <red_envelop>`.
 
-  - Is it working as expected? In case the email is sent using Odoo, the bounce email should reach
-    the Odoo database and display the :ref:`red envelope <red_envelop>`.
+    .. seealso::
+       The bounce system parameter needs to be set in the technical settings in order for the
+       database to correctly receive bounce messages. To access this setting go to:
+       :menuselection:`Settings app --> Technical --> Parameters --> System Parameters`. The
+       parameter name to select is `mail.bounce.alias`.
```

Comment:
This seealso is indented too far, please scoot it back and add a :guilabel: to the element with active instruction :)
```suggestion
  .. seealso::
      The bounce system parameter needs to be set in the technical settings in order for the
      database to correctly receive bounce messages. To access this setting go to:
      :menuselection:`Settings app --> Technical --> Parameters --> System Parameters`. The
      parameter name to select is :guilabel:`mail.bounce.alias`.
```